source/features/copy-file.tsx

```diff
@@ -5,16 +5,22 @@ import copyToClipboard from 'copy-text-to-clipboard';
 import features from '../libs/features';
 
 function handleClick({currentTarget: button}: React.MouseEvent<HTMLButtonElement>): void {
-	const file = button.closest('.Box');
+	const selector = features.isGist() ? '.js-gist-file-update-container' : '.Box';
+	const file = button.closest(selector);
 	const content = select.all('.blob-code-inner', file!)
 		.map(({innerText: line}) => line === '\n' ? '' : line) // Must be `.innerText`
 		.join('\n');
 	copyToClipboard(content);
 }
 
 function renderButton(): void {
-	for (const blameButton of select.all('[data-hotkey="b"]')) {
-		blameButton
+	const selector = features.isGist() ? '.file-actions .btn' : '[data-hotkey="b"]';
+	for (const button of select.all(selector)) {
+		if (features.isGist()) {
+			button.classList.add('BtnGroup-item');
```

Comment:
You can use the groupButtons function to do this automatically, without conditions. Call it after the prepend

Reply:
Have I done it right?

Reply:
You can leave the old code (without the extra const) and use `groupSiblings(button)` instead, I think.

Reply:
I thought you were literally talking `groupButtons` function.

Reply:
Sorry about the confusion 😁